### PR TITLE
fix: remove dependencies to prevent infinite rerenders

### PR DESCRIPTION
**Pull request by EsterB12**

### CODE REVIEW

File: components/nft-sale-widget/hooks/useMintSection.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export const useMintSection = (isRandom: boolean) => {
     };
 
     void fetchWhitelist();
-  }, [activeSale, getAddresses, isAddressesLoading]);
```
Comment (williamslsy) [inline components/nft-sale-widget/hooks/useMintSection.ts:92]: yeah `getAddresses` here might be technically unnecessary as a dependency but that's not the cause of rerendering

File: hooks/useImageData.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const useImageData = (tokenIds: string[]) => {
     } else {
       setImageUris(Array(tokenIds.length).fill(FALLBACK_IMAGE));
     }
-  }, [data, tokenIds]);
```
Comment (williamslsy) [inline hooks/useImageData.tsx:27]: this seems to be the source of the re-rendering. so the tokenIds dependency can be removed or better still memoized

**Review by williamslsy — CHANGES_REQUESTED** [review]

File: components/nft-sale-widget/hooks/useMintSection.ts (hunk 1 above)
Comment (EsterB12) [reply components/nft-sale-widget/hooks/useMintSection.ts:92]: resolved it as we discussed! :)

**Comment by EsterB12** [comment]
Closed because this forms part of #156

**Pull request closed without merging**